libs/executors/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2020 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+# Default location is $HPX_ROOT/libs/executors/include
+set(executors_headers
+  hpx/execution/executors/distribution_policy_executor.hpp
+  hpx/execution/executors/current_executor.hpp
+  hpx/execution/executors/guided_pool_executor.hpp
+  hpx/execution/executors.hpp
+  hpx/execution/executors/limiting_executor.hpp
+  hpx/execution/executors/parallel_executor_aggregated.hpp
+  hpx/execution/executors/parallel_executor.hpp
+  hpx/execution/executors/pool_executor.hpp
+  hpx/execution/executors/restricted_thread_pool_executor.hpp
+  hpx/execution/executors/sequenced_executor.hpp
+  hpx/execution/executors/service_executors.hpp
+  hpx/execution/executors/thread_pool_attached_executors.hpp
+  hpx/execution/executors/timed_executors.hpp
```

Comment:
Should we go with just `hpx/executors/*_executor.hpp` now? This seems like it'll be the final stop for the executors.

Reply:
I'd support that.

Reply:
Yep I'll do that, I just have to fix the current circular dependency :)